colossalai/auto_parallel/solver/sharding_strategy.py

```diff
@@ -37,6 +37,45 @@ class ShardingStrategy:
     input_shardings: List[ShardingSpec] = None
 
 
+@dataclass
+class TrainCycleItem:
+    """
+    TrainCycleItem is a dataclass to store the items which have different values for the forward and backward pass
+    in a training iteration.
+
+    Args:
+        fwd (Any): the item for the forward pass
+        bwd (Any): the item for the backward pass
+    """
+    fwd: Any
+    bwd: Any
+
```

Comment:
I think we could add a 'total' attribute into TrainCycleItem, I find it is useful during preparing costs information for solver.

Reply:
ok